src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Automation/Peers/MenuItemAutomationPeer.cs

```diff
@@ -21,13 +21,18 @@
 namespace System.Windows.Automation.Peers
 {
     ///
-    public class MenuItemAutomationPeer : FrameworkElementAutomationPeer, IExpandCollapseProvider, IInvokeProvider, IToggleProvider
+    public class MenuItemAutomationPeer : ItemsControlAutomationPeer, IExpandCollapseProvider, IInvokeProvider, IToggleProvider
```

Comment:
This is a break API changed. I think it's worth discussing here.

Reply:
Unfortunately there is now way around this. What are the consequences?

Reply:
@RonHeck Maybe this PR will be slowly merged.

Reply:
That's Ok. Finally I hope it may solved somewhere in the future. May it can be merged to .NET7 as it is pretty fresh.

Reply:
@lindexi why is this a breaking API change?